index.js

```diff
@@ -188,7 +193,17 @@ module.exports = {
             const groupKey = lodashGet(msg, 'value.content.groupKey')
             const groupId = lodashGet(msg, 'value.content.recps[0]')
 
-            ssb.box2.addGroupInfo(groupId, { key: groupKey, root: groupRoot })
+            ssb.box2.getGroupKeyInfo(groupId, (err, info) => {
+              if (err) throw err
```

Comment:
This `throw` could be dangerous (could crash the app). I suggest you use a console.error instead.